src/Resolver/AbstractInjection.php

```diff
@@ -20,6 +28,11 @@
      */
     public function setParameterName(string $name) : self
     {
+        trigger_error(
```

Comment:
To be documented in `CHANGELOG.md`

Reply:
Cahngelog updates will follow. this is still WIP